bundles/org.openhab.binding.neeo/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -10,4 +10,11 @@
 		on the Devices</description>
 	<connection>local</connection>
 
+	<discovery-methods>
```

Comment:
The neeo transport could be discovered in the same way.

Reply:
Ok. Done.